ifrs17/Import/ImportScopeCalculation.ipynb

```diff
@@ -2053,26 +2053,27 @@
-                "\n                                                                                                                                          AccidentYear = x.Identity.AccidentYear,",
-                "\n                                                                                                                                          AmountType = x.Identity.AmountType,",
-                "\n                                                                                                                                          Value = x.Value,",
-                "\n                                                                                                                                          Partition = GetStorage().TargetPartition",
-                "\n                                                                                                                                        });",
-                "\n    IEnumerable<IfrsVariable> PvCurrent => GetScope<PvCurrent>(Identity).PresentValues.Where(x => Math.Abs(x.Value) >= Precision).Select(x => new IfrsVariable{ EconomicBasis = x.EconomicBasis, ",
-                "\n                                                                                                                                          EstimateType = x.Identity.EstimateType, ",
-                "\n                                                                                                                                          DataNode = x.Identity.Id.DataNode, ",
-                "\n                                                                                                                                          AocType = x.Identity.Id.AocType, ",
-                "\n                                                                                                                                          Novelty = x.Identity.Id.Novelty, ",
-                "\n                                                                                                                                          AccidentYear = x.Identity.AccidentYear,",
-                "\n                                                                                                                                          AmountType = x.Identity.AmountType,",
-                "\n                                                                                                                                          Value = x.Value,",
-                "\n                                                                                                                                          Partition = GetStorage().TargetPartition",
-                "\n                                                                                                                                        });",
+                "\n    IEnumerable<IfrsVariable> PvLocked => GetScope<PvLocked>(Identity).PresentValues.Select(x => ",
```

Comment:
I prefer previous indentation. It also ease the review. Could you go back? thanks

Reply:
Properties weren't indented at all, and with the removal of the Where filter the format of these cells becomes even worse. A refactor of the indentation was necessary.